polar-core/src/partial/partial.rs

```diff
@@ -325,17 +325,25 @@ mod test {
     }
     macro_rules! assert_partial_binding {
         ($bindings:expr, $sym:expr, $($args:expr),+) => {
-            assert_eq!(
-                $bindings
+            let l = $bindings
                     .get(&sym!($sym))
                     .expect(&format!("{} is unbound", $sym))
                     .value()
                     .as_expression()
                     .unwrap()
                     .clone()
                     .into_iter()
-                    .collect::<HashSet<Term>>(),
-                    hashset! { $($args),+ })
+                    .collect::<HashSet<Term>>();
+            let r = hashset! { $($args),+ };
+            let fmt = |hs: &HashSet<Term>| {
```

Comment:
Nit: this could be:

```rust
let fmt = |hs: &HashSet<Term>| format!("{{ {} }}", hs.iter().map(|t| t.to_polar()).collect::<Vec<String>>().join(", "));